Mata/Emit/MapEmitDebugInfo.cs

```diff
@@ -22,8 +24,9 @@ public class MapEmitDebugInfo<T>
 
         private int line = 1;
 
+#if !NETSTANDARD1_6
         private ISymbolDocumentWriter symbolDocumentWriter;
-
+#endif
         public MapEmitDebugInfo(string typeNamespace, string typeName, string className)

```

Comment:
This entire class is not needed if the symbol document writer is not available.  Lets wrap this entire class in an #if def so it just goes away, rather than litter the code with smaller scoped #if defs